Storage/DefaultSuggestedSites.swift

```diff
@@ -0,0 +1,28 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+
+public class DefaultSuggestedSites {
+    public static let sites = [
+        SuggestedSiteData(
+            url: NSLocalizedString("https://www.mozilla.org/about", comment: "Tile url for Mozilla"),
```

Comment:
Dont' localize the URLs. Just the titles. The URL will auto-redirect to the correct locale. There is nothing to localize for the L10N team there.